dt/uni04delta/networking/netconfig/kustomization.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+patches:
+  - target:
+      kind: NetConfig
```

Comment:
I'm wondering if this `kustomization` is run because I see:

```
[zuul@controller-0 ~]$ oc get openstackdataplanenodeset
NAME         STATUS   MESSAGE
ceph-nodes   False    NodeSetIPReservationReady error occurred admission webhook "vipset.kb.io" denied the request: IPSet.network.openstack.org "ceph-2" is invalid: spec.networks[2].name: Invalid value: "nfs": network nfs not in NetConfig
```
and `NetConfig` does not contain the NFS network.

Reply:
I can confirm that after manually editing `NetConfig` adding  the missing network the dataplane converged and I can see it configured in the Ceph nodes.
However, I think there are some issues as  I don't see nfs among the  `net-attach-def` and `nncp` on the OCP nodes.
@fultonj @abays anything obvious you see here?

Reply:
I see that this is a new file added in this PR.  But I also notice that we already have `NetConfig` `patches` elsewhere in the DT [1].  I think we should consolidate them into one location.  Here or the original location seems fine.  But also I think the patch here needs `name: netconfig` added to it so that it can properly find the target resource, like we were already doing with the existing patch for `storagemgmt` network [2].

[1] https://github.com/openstack-k8s-operators/architecture/blob/f09f1b4ade5dc5b440d5665510bab165ae0b3e42/dt/uni04delta/kustomization.yaml#L25-L70
[2] https://github.com/openstack-k8s-operators/architecture/blob/f09f1b4ade5dc5b440d5665510bab165ae0b3e42/dt/uni04delta/kustomization.yaml#L29C1-L29C22

Reply:
@abays thanks for pointing that out. 

The latest update removes `dt/uni04delta/networking/netconfig/kustomization.yaml` and updates `dt/uni04delta/kustomization.yaml` to make the patches to the network named nfs the same way they are made to the network named storagemgmt.